cognee/infrastructure/llm/structured_output_framework/litellm_instructor/llm/mistral/adapter.py

```
@@ -0,0 +1,129 @@
+            - system_prompt (str): The system prompt that will be shown alongside the user input.
+
+        Returns:
+        --------
+            - str: The formatted prompt string combining system prompt and user input.
+        """
+        if not text_input:
+            text_input = "No user input provided."
+        if not system_prompt:
+            raise MissingSystemPromptPathError()
+
+        system_prompt = LLMGateway.read_query_prompt(system_prompt)
```

Pylint (4.0.1)

[error] Class 'LLMGateway' has no 'read_query_prompt' member

(E1101)